uPortal-webapp/src/main/resources/properties/security.properties

```diff
@@ -76,6 +76,13 @@ cas.authenticationFilter.cas.login.url=${cas.protocol.server.context}/login
 #cas.ticketValidationFilter.proxyReceptorUrl=/CasProxyServlet
 #cas.ticketValidationFilter.ticketValidator.proxyCallbackUrl=${portal.protocol}://${portal.lbServerName}${portal.context}${cas.ticketValidationFilter.proxyReceptorUrl}
 
+## Some CAS servers, like the CAS server in uPortal-start can not handle encoded service URLs.
+## Set the following property to false to disable encoding of service URLs.
+## See https://groups.google.com/a/apereo.org/d/msg/uportal-user/44Uw1YP8_Mg/hLaTlEVZFAAJ
+## for the discussion regarding this property
+#
+#cas.ticketValidationFilter.encodeServiceUrl=true
+
```

Comment:
Once this is merged, we'll set this to false in uPortal-start's `uPortal.properties`